includes/class-wc-stripe-upe-compatibility.php

```diff
@@ -15,4 +15,18 @@ public static function is_wp_supported() {
 	public static function is_wc_supported() {
 		return version_compare( WC_VERSION, self::MIN_WC_VERSION, '>=' );
 	}
+
+	public static function are_inbox_notes_supported() {
```

Comment:
We no longer need `version_compare( WC_VERSION, '4.4.0', '>='` because this function now checks the `WC_Data_Store` directly? I think that's good.

Reply:
Yes, that's correct 👍